core/controllers/reader.py

```diff
@@ -183,9 +184,9 @@ def _get_exploration_player_data(
             interaction_templates),
         'is_private': rights_manager.is_exploration_private(
             exploration_id),
-        # Note that this overwrites the value in base.py.
+        # Note that this overwrites the value in base_classifier.py.
```

Comment:
I don't think this and the next line have anything to do with classifiers. base.py is the correct thing here.